api/src/pcapi/core/fraud/models.py

```diff
@@ -312,7 +386,9 @@ class BeneficiaryFraudReview(PcObject, Model):
 @dataclasses.dataclass
 class FraudItem:
     status: FraudStatus
-    detail: typing.Optional[str]
+    # The detail field should not be optional because a OK status alone does
+    # not provide any information about the validation done
```

Comment:
on mettra quoi comme "information about the validation done" ?
dès qu'on s'est mis d'accord et qu'on fait ça partout on pourra enlever le comm

Reply:
J'ai déjà mis un texte de `detail` pour tous les `FraudItem` OK, c'est dans la PR.
Quelques exemple:

- `f"Ubble score {content.score}"`
- `"Le numéro de pièce d'identité est valide"`
- `"Utilisateur non dupliqué"`
- etc...

Il s'agit simplement d'indiquer quel vérification a été faite et le fait qu'elle est OK,  pense pas que ça nécéssite tant de discussion pour se mettre d'accord